discovery-provider/src/tasks/entity_manager/entity_manager.py

```diff
@@ -210,17 +219,21 @@ def entity_manager_update(
 
                 # invalidate all new records except the last
                 for record in records:
-                    record.is_current = False
+                    if "is_current" in record.__dict__:
```

Comment:
is this fixing a bug, or a specific add for this notif change?

Reply:
i think this is just because the notification table doesn't have is_current so no need to invalidate like we do for other entities